doc/cli/npm-doctor.md

```diff
@@ -0,0 +1,17 @@
+npm-doctor(1) -- Check your environments
+========================================================
+
+## SYNOPSIS
+
+    npm doctor
+
+## DESCRIPTION
+
+This command helps you when you get struck by environments which
+potentially have a problem. Some of the questions are same with
+we ask in the ISSUE_TEMPLATE.md.
```

Comment:
I think the best way to do that would be to template it, but I am not sure about what anyone's thoughts are about including a template engine with `npm`.